### ISSUE TITLE
Numbering levels do not apply when called as instance methods (static required)

### ISSUE THREAD

**Issue by jamesmontalvo3** [dolanmiu/docx#389]
Numbering does not appear to be responding to what is defined in `createLevel` method. See the screenshot below, which is generated by the code that follows.

![image](https://user-images.githubusercontent.com/716482/63727064-ccb06180-c824-11e9-9531-48acbcf904a4.png)

Adjusting the indent levels or list type (e.g. `upperRoman` vs `decimal`) in `createLevel()` has no effect. I tested this in three different ways, commented as `OPTION 1`, `OPTION 2`, `OPTION 3` in the code below. In each option the output looks the same.

```js
var fs = require('fs')
var docx = require('docx');

const doc = new docx.Document();

const numbering = new docx.Numbering();

const abstractNum = numbering.createAbstractNumbering();

// OPTION 1: No createLevel

// OPTION 2:
// abstractNum.createLevel(0, "upperRoman", "%1", "start").addParagraphProperty(new docx.Indent(360, 130));
// abstractNum.createLevel(1, "decimal", "%2", "start").addParagraphProperty(new docx.Indent(1200, 3000));
// abstractNum.createLevel(2, "lowerLetter", "%3	", "start").addParagraphProperty(new docx.Indent(4000, 5000));

// OPTION 3;
// abstractNum.createLevel(0, "upperRoman", "%1", "start").indent({ left: 360, hanging: 130 });
// abstractNum.createLevel(1, "decimal", "%1.%2.", "start").indent({ left: 1200, hanging: 3000 });
// abstractNum.createLevel(2, "lowerLetter", "%3)", "start").indent({ left: 4000, hanging: 5000 });

const concrete = numbering.createConcreteNumbering(abstractNum);

doc.addSection({
	children: [
		new docx.Paragraph({ text: "Item 1", numbering: { num: concrete, level: 0 }}),
		new docx.Paragraph({ text: "Item 1.1", numbering: { num: concrete, level: 1 }}),
		new docx.Paragraph({ text: "Item 1.2", numbering: { num: concrete, level: 1 }}),
		new docx.Paragraph({ text: "Item 1.2.1", numbering: { num: concrete, level: 2 }}),
		new docx.Paragraph({ text: "Item 1.2.2", numbering: { num: concrete, level: 2 }}),
		new docx.Paragraph({ text: "Item 1.2.3", numbering: { num: concrete, level: 2 }}),
		new docx.Paragraph({ text: "Item 1.3", numbering: { num: concrete, level: 1 }}),
		new docx.Paragraph({ text: "Item 2", numbering: { num: concrete, level: 0 }}),
		new docx.Paragraph({ text: "Item 3", numbering: { num: concrete, level: 0 }}),
	],
});

docx.Packer.toBuffer(doc).then((buffer) => {
	fs.writeFileSync("docx-simple-num.docx", buffer);
});
```

Using docx version `5.0.0-rc5` and Node `v10.16.3`

**Comment by dannycornelisseMT** [comment]
I have the exact same problem

**Comment by jamesmontalvo3** [comment]
I've been trying to troubleshoot this and identify where the problem started. In doing so, I noticed that the RunKit specified in the README for [4.0.0](https://runkit.com/dolanmiu/docx-demo3/4.0.0) as well as [1.0.0](https://runkit.com/dolanmiu/docx-demo3/1.0.0) are exhibiting these issues as well (i.e. not showing `upperRoman` or `lowerLetter` list types). I tested 5.x, 4.x, and 3.x versions and found this issue everywhere, but then realized that [3.6.0/demo/demo2.js](https://github.com/dolanmiu/docx/blob/3.6.0/demo/demo2.js) _does work_ (i.e. displays list as `lowerLetter`), but [3.6.0/demo/demo3.js](https://github.com/dolanmiu/docx/blob/3.6.0/demo/demo3.js) _does not_ (displays default `decimal`). Following `demo2.js` as an example, I applied the following diff to [3.6.0/demo/demo3.js](https://github.com/dolanmiu/docx/blob/3.6.0/demo/demo3.js) to make it work:

```diff
 var doc = new docx.Document();

-const numbering = new docx.Numbering();
+const abstractNum = doc.Numbering.createAbstractNumbering();

-const abstractNum = numbering.createAbstractNumbering();
 abstractNum.createLevel(0, "upperRoman", "%1", "start")
     .addParagraphProperty(new docx.Indent(720, 260));
 abstractNum.createLevel(1, "decimal", "%2.", "start")
     .addParagraphProperty(new docx.Indent(1440, 980));
 abstractNum.createLevel(2, "lowerLetter", "%3)", "start")
     .addParagraphProperty(new docx.Indent(2160, 1700));

-const concrete = numbering.createConcreteNumbering(abstractNum);
+const concrete = doc.Numbering.createConcreteNumbering(abstractNum);

 var topLevelP = new docx.Paragraph("Hey you");
 var subP = new docx.Paragraph("What's up fam");
```

I applied this same fix to the RunKit demo [here](https://runkit.com/jamesmontalvo3/docx-demo-3), and it works. Also works locally with 5.x.

In summary, it appears to be working if `createAbstractNumbering()` and `createConcreteNumbering()` are called as static vs instance methods.

**Comment by jamesmontalvo3** [comment]
Renamed issue to account for the fact that it is possible to alter numbering format, bu only if done as described in the comment above. Additionally I'll open a separate issue for the paragraph styling problem ([comment 2](https://github.com/dolanmiu/docx/issues/389#issuecomment-525062692)).

**Comment by dolanmiu** [comment]
API is now changed, please refer to:

https://github.com/dolanmiu/docx/blob/master/demo/3-numbering-and-bullet-points.ts
https://github.com/dolanmiu/docx/blob/master/demo/29-numbered-lists.ts
https://github.com/dolanmiu/docx/blob/master/demo/2-declaritive-styles.ts
